Docs/sphinx_documentation/source/Visualization.rst

```diff
@@ -79,16 +102,20 @@ you can build into a pdf using pdflatex).
    :math:`\rightarrow` "Dataset" in order to look at the actual numerical
    values (see :numref:`Fig:Amrvis`).  Or you can simply left
    click on a point to obtain the numerical value.  You can also export the
-   pictures in several different formats under "File/Export".  In 2D you can
-   right and center click to get line-out plots.  In 3D you can right and
-   center click to change the planes, and the hold shift+(right or center)
+   pictures in several different formats under ``File/Export``.  In 2D you can
+   right or center click to get line-out plots.  In 3D you can right or
+   center click to change the planes, and hold shift+(right or center)
    click to get line-out plots.
 
-   We have created a number of routines to convert AMReX plotfile data other
+   |
+
+   We have created a number of routines to convert AMReX plotfile data to other
    formats (such as matlab), but in order to properly interpret the
```

Comment:
Capitalize correctly? (Not that is was before :) )

Reply:
Yup :+1: